### PR TITLE
chore: Add Hindi translation

### PR DESCRIPTION
**Pull request by NayakNavin**
I have added Hindi translation - #13

### CODE REVIEW

File: app/src/main/res/values-hi-rIN/strings.xml [hunk 1]
```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <string name="app_name">नोट क्रिप्ट प्रो</string>
```
Comment (RyuzakiKK) [inline app/src/main/res/values-hi-rIN/strings.xml]: Please do not translate the app name here, and also where "Note Crypt" is mentioned in the other strings

**Review by RyuzakiKK — CHANGES_REQUESTED** [review]
Thank you!
Please do not translate every note crypt occurrences and then it should be perfect.

**Comment by NayakNavin** [comment]
I've updated the translations as you suggested.

**Review by RyuzakiKK — APPROVED** [review]

**Comment by RyuzakiKK** [comment]
Perfect.

**Pull request merged**